docs/management/connectors/action-types/servicenow-sir.asciidoc

```diff
@@ -1,25 +1,83 @@
 [role="xpack"]
 [[servicenow-sir-action-type]]
-=== ServiceNow connector and action
+=== ServiceNow SecOps connector and action
 ++++
 <titleabbrev>ServiceNow SecOps</titleabbrev>
 ++++
 
-The ServiceNow SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow security incidents.
+The {sn} SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create {sn} security incidents.
+
+[float]
+[[servicenow-sir-connector-prerequisites]]
+==== Prerequisites
+After upgrading from {stack} version \<=7.15.0 to version >=7.16.0, you must complete the following within your {sn} instance before creating a new {sn} SecOps connector or <<servicenow-sir-connector-update, updating an existing one>>:
+
+* Install https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f[Elastic for Security Operations (SecOps)] from the {sn} Store.
+* Create a {sn} integration user and assign it the appropriate roles.
+* Create a Cross-Origin Resource Sharing (CORS) rule.
+
+*Create a {sn} integration user*
+
+To ensure authenticated communication between Elastic and {sn}, create a {sn} integration user and assign it the appropriate roles. 
+
+. In your {sn} instance, go to *System Security -> Users and Groups -> Users*.
+. Click *New*.
+. Complete the form, then right-click on the menu bar and click *Save*.
+. Go to the *Roles* tab and click *Edit*.
+. Assign the integration user the following roles: 
+* `import_set_loader`
+* `import_transformer`
+* `personalize_choices`
+* `sn_si.basic`
+* `x_elas2_sir_int.integration_user`
+. Click *Save*.
+
+*Create a CORS rule*
+
+A CORS rule is required for communication between Elastic and {sn}. To create a CORS rule:
```

Comment:
Not sure you need the sentence "To create a CORS rule:"

Reply:
I've wondered about this as well because the lead-in sentence essentially repeats what the sub-section title conveys. I'm partial to leaving it as is though, just so it matches the way these instructions were written in the Security docs. WDTY?

Reply:
sounds good to me.